README.md

```diff
@@ -15,3 +15,32 @@ The following services are part of the data access services (`data-access-servic
 The `data-access-services` sub-project is a multi-module Maven project which includes a library module (`core-banking-lib`) for sharing configuration and code between the data access services.
 
 TODO: Once the data schema and names have been decided this list needs to be updated. Currently, it only includes the transaction service for showing how the multi-module Maven project is set up. Therefore, when adding new services most of the `pom.xml` of the transaction service can be taken over.
+
+# Development
+
+This section describes how to get started with development for this repository.
+
+## Required software
+
+- Git
+- JDK 11
+- TBD
+- IntelliJ IDEA is recommended for Java development
+
+## Accessing the source code
+
+The code for this project is hosted in the public github repository:
+https://github.com/DeloitteDigitalAT/core-banking.git
+
+The repo can be cloned using
+- SSH key: `git clone git@github.com:DeloitteDigitalAT/core-banking.git`
+- github user/pwd (deprecated):
+`git clone https://github.com/DeloitteDigitalAT/core-banking.git`
+
+
+## Configure static code analysis tools
+- Checksyle: used to enforce Java coding convenions and formatting specified in `/config/checkstyle/checkstyle.xml`.
+Configuration guide is available at https://github.com/DeloitteDigitalAT/core-banking/blob/master/config/checkstyle/README.md
+- Sonar
+- TBD
```

Comment:
Same as above

Reply:
some details should be added here about Sonar + JS lint later